org-clubhouse.el

```diff
@@ -665,9 +665,10 @@ If the epics already have a CLUBHOUSE-EPIC-ID, they are filtered and ignored."
                          (number-to-string story-id)))
 
       (org-set-property "clubhouse-epic"
-                        (org-make-link-string
-                         (org-clubhouse-link-to-epic epic-id)
-                         (alist-get epic-id (org-clubhouse-epics))))
+        (if epic-id (org-make-link-string)
+          (org-clubhouse-link-to-epic epic-id)
+          (alist-get epic-id (org-clubhouse-epics))
+          "None"))
```

Comment:
@glittershark the formatting is obviously wrong. I was wondering which tool you are using to format+config your code. Parinfer on my side wants to reformat everything.